commit c6ccec336987780358d316351cab4b6b77c506e2
Author: Albert Astals Cid <aacid@kde.org>
Date:   Sun Mar 3 23:55:42 2013 +0100

    Load the correct catalog
    
    It'd be cool if you guys read the comments to your bugs
    BUGS: 307891
    FIXED-IN: 4.10.2
    CCMAIL: plasma-devel@kde.org

diff --git a/plasma/desktop/shell/activitymanager/activitymanager.cpp b/plasma/desktop/shell/activitymanager/activitymanager.cpp
index 24726d4..0a515f2 100644
--- a/plasma/desktop/shell/activitymanager/activitymanager.cpp
+++ b/plasma/desktop/shell/activitymanager/activitymanager.cpp
@@ -99,7 +99,7 @@ void ActivityManagerPrivate::init(Plasma::Location loc)
     structure->setPath(path);
 
     package = new Plasma::Package(path, pluginName, structure);
-    KGlobal::locale()->insertCatalog(pluginName);
+    KGlobal::locale()->insertCatalog("plasma_package_org.kde.desktop.activitymanager");
 
     declarativeWidget = new Plasma::DeclarativeWidget(q);
     declarativeWidget->setInitializationDelayed(true);
